nservicebus/architecture/publishing-from-web-applications.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Publishing Events from Web Applications
+summary: "Guidance on why you should or should not publish events from a web application"
+tags: []
+redirects:
+---
+
+You may have heard that [it's bad practice to publish events from a web application](http://www.make-awesome.com/2010/10/why-not-publish-nservicebus-messages-from-a-web-application/). This guidance is a bit outdated, as improvements Microsoft has made to IIS, more varied methods of hosting .NET web applications, and the addition of additional NServiceBus transports have made things much more nuanced.

```

Comment:
I wrestled with that. Having a reciprocal link doesn't hurt SEO for either page, given that page on my blog comes up first on a bunch of different searches. In my mind. having the link prevents people from going "Wait, but I thought..." because it illustrates that before, X, and now, X agrees that the guidance is Y. But we could take that chunk out for sure. What do you think?

Either way my blog post will be changed to link to the official doco right at the top.